package.json

```
@@ -24,20 +20,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mozilla/firefox-launch.git"
+    "url": "git+https://github.com/mozilla-extensions/firefox-bridge"
   },
-  "keywords": [
-    "firefox",
-    "addon",
-    "chrome",
-    "edge"
-  ],
-  "author": "mozilla",
-  "license": "ISC",
+  "author": "s/mozilla/Mozilla",
```

```suggestion
  "author": "Mozilla",
```

Sorry for the confusion, "s/<old>/<new>" is an ancient term I use to say "replace <old> with <new>".